test-plan.md

```diff
@@ -0,0 +1,98 @@
+# Validating C# Extension for VS Code
+
+#### Opening projects
+When you open a directory in VS Code, the C# extension should look for a .csproj or .sln file in that directory and use "OmniSharp" to load it. 
+If you look in "Output > Omnisharp Log" a bunch of information should be printed about what copy of MSBuild was used and what projects were load
+
+Project types to test:
+* Standalone csproj 
+* Directory containing .sln file that references csprojs--projects should be loaded 
+* .NET Core/.NET Standard csproj
+* (Windows) Desktop .NET projects
```

Comment:
We always get a bunch of feedback on Unity projects. Should we add something there too?

Reply:
Yes. I've typically validated Unity projects.

Reply:
Sure--do you have any examples of unity projects people edit?

Reply:
I sent some pointers in email.